src/ReactiveDomain/ReactiveDomain.Tests/ReactiveDomain.Tests.csproj

```diff
@@ -63,6 +63,10 @@
       <HintPath>..\..\..\packages\Newtonsoft.Json.9.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="ReactiveUI, Version=6.5.0.0, Culture=neutral, processorArchitecture=MSIL">

```

Comment:
Is this needed? I don't see any references to it from any of the new ReactiveDomain.Tests classes.

Reply:
I removed it and it builds. it may be that I took out whatever used it, but never removed the reference.